go.mod

```diff
@@ -0,0 +1,9 @@
+module api-booking-time
```

Comment:
Normally, a module is named after the path it follows in your code repository.

In this case, the suggestion is to rename it as:
```suggestion
module github.com/bornehill/golang-bootcamp-2020
```